### DISCUSSION TITLE
Custom command for stock Windows 11 Start Menu?

### DISCUSSION THREAD

**Discussion by Hybrid-Divide** [Open-Shell/Open-Shell-Menu#1985] [Q&A] [answered]
I know in the Main Menu options, you can check an option to "Show Start screen shortcut", to have a button at the top of your menu to open the stock Win11 start menu.

Sadly, I don't want it there.  I want it in a submenu.  But OpenShell won't let me drag that button around!

Is there a custom command I can assign to a "CUSTOM" item in the "Customize Start Menu" settings to launch the stock start menu that way?  (I know I can just hit Shift + Windows Key to launch it, but I'd like to be able to do so via a custom button, too!

(See the screenshot to see where I'd like to put it.  Currently, I just have the links to the Programs and Apps menus from there.)

Thanks!
![Screenshot 2024-09-01 234236](https://github.com/user-attachments/assets/8bed2284-0786-400d-9895-2ea460d6b04d)

**Comment by ge0rdi** [comment, marked as answer]
Right-click on `Start Menu (Windows)` item and select `Properties`.
You will see that it just starts `StartMenu.exe` with `-togglenew` parameter.

You can easily create custom command that does the same thing.

**Reply by Hybrid-Divide** [reply]
Perfect! Thank you!
